### ISSUE TITLE
The PCD obtained by velodyne16 does not seem to yield good voxels

### ISSUE THREAD

**Issue by af-doom** [hku-mars/mlcc#23]
![2023-07-11 19-42-32 的屏幕截图](https://github.com/hku-mars/mlcc/assets/91470350/b44cf19d-81e7-4a0e-aa44-860f2bbdc5f4)

**Comment by samsdolphin** [comment]
Hi @af-doom, I apologize for my late reply. Have you solved this issue? Can you share the point cloud, image, and rosbag so I can help you to debug it?

**Comment by af-doom** [comment]
Thank you very much for your reply. The image, bag, and PCD data are attached. Thank you very much
test_data.zip


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "hku-mars/mlcc"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月11日(星期一) 晚上11:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [hku-mars/mlcc] The PCD obtained by velodyne16 does not seem to yield good voxels (Issue #23)





 
Hi @af-doom, I apologize for my late reply. Have you solved this issue? Can you share the point cloud, image, and rosbag so I can help you to debug it?

**Comment by samsdolphin** [comment]
> Thank you very much for your reply. The image, bag, and PCD data are attached. Thank you very much test_data.zip
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "hku-mars/mlcc" ***@***.***&gt;; 发送时间:&nbsp;2023年9月11日(星期一) 晚上11:54 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [hku-mars/mlcc] The PCD obtained by velodyne16 does not seem to yield good voxels (Issue #23) Hi @af-doom, I apologize for my late reply. Have you solved this issue? Can you share the point cloud, image, and rosbag so I can help you to debug it? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***&gt;

Hi @af-doom, I can't find your attached files here. Can you double-check?

**Comment by af-doom** [comment]
I'm very sorry, could you try this one

https://chdeducn-my.sharepoint.com/:u:/g/personal/2018901230_chd_edu_cn/EdmgF7ZRwPZClsj9YYD04noBr5z5VAaVb_P0remBl7ZrZw?e=X3SjHY







---Original---
From: "Xiyuan ***@***.***&gt;
Date: Fri, Sep 15, 2023 16:02 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hku-mars/mlcc] The PCD obtained by velodyne16 does not seem toyield good voxels (Issue #23)




  
Thank you very much for your reply. The image, bag, and PCD data are attached. Thank you very much test_data.zip
 …
 ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "hku-mars/mlcc" @.&gt;; 发送时间:&nbsp;2023年9月11日(星期一) 晚上11:54 @.&gt;; @.@.&gt;; 主题:&nbsp;Re: [hku-mars/mlcc] The PCD obtained by velodyne16 does not seem to yield good voxels (Issue #23) Hi @af-doom, I apologize for my late reply. Have you solved this issue? Can you share the point cloud, image, and rosbag so I can help you to debug it? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: @.***&gt;
  
Hi @af-doom, I can't find your attached files here. Can you double-check?

**Comment by lsangreg** [comment]
Hi, how did you resolve?

**Comment by Chatoyant19** [comment]
I also have this problem.Did you solve it?

**Comment by lsangreg** [comment]
@Chatoyant19  tbh i wasn't able to reproduce their results  with their data, what about you?

**Comment by Chatoyant19** [comment]
> @Chatoyant19 tbh i wasn't able to reproduce their results with their data, what about you?

I can get a good result with their data.I think it is necessary  to adjust some parameters, when using myself datas. Because after I changed 'voxel_size' and 'eigen_ratio', I can extract more useful planes.But it is not enough, for example, the same wall cannot be extracted completely when the plane is extracted.

**Comment by lsangreg** [comment]
> > @Chatoyant19 tbh i wasn't able to reproduce their results with their data, what about you?
> 
> I can get a good result with their data.I think it is necessary to adjust some parameters, when using myself datas. Because after I changed 'voxel_size' and 'eigen_ratio', I can extract more useful planes.But it is not enough, for example, the same wall cannot be extracted completely when the plane is extracted.

I think at least it is important to reproduce what they say " the average translation and rotation errors down to 6mm and 0.09 degrees for LiDAR-camera" , which for me is more or less angular error 0.067 degree baseline error 0.025 m ( should be 0.006).

With this being said I will also try to investigate more eigen and voxel size... I was working more on the edge detection intensity as my data are synthetic and highly defined...

Can I ask you which did error you introduce to your initial extrinsics?

**Comment by lsangreg** [comment]
@Chatoyant19  did you obtain better results?

**Comment by Chatoyant19** [comment]
> @Chatoyant19 did you obtain better results?

No, I didn't. I know that my problem lies in extracting planar features from the LiDAR point clouds. I 'm unable to extract enough and high-quality planar features.Maybe use PCA to extract planar features is closely related to the quality of point cloud mapping? My LiDAR model is the Hesai FT120, which has relatively poor ranging accuracy.

**Comment by af-doom** [comment]
> > @Chatoyant19 did you obtain better results?
> 
> No, I didn't. I know that my problem lies in extracting planar features from the LiDAR point clouds. I 'm unable to extract enough and high-quality planar features.Maybe use PCA to extract planar features is closely related to the quality of point cloud mapping? My LiDAR model is the Hesai FT120, which has relatively poor ranging accuracy.

Hello, do you have data about FT120 and camera? Can you share it with me for testing?

**Comment by Chatoyant19** [comment]
> > > @Chatoyant19 did you obtain better results?
> > 
> > 
> > No, I didn't. I know that my problem lies in extracting planar features from the LiDAR point clouds. I 'm unable to extract enough and high-quality planar features.Maybe use PCA to extract planar features is closely related to the quality of point cloud mapping? My LiDAR model is the Hesai FT120, which has relatively poor ranging accuracy.
> 
> Hello, do you have data about FT120 and camera? Can you share it with me for testing?

Of course, can I have your email address? I'll send you the relevant data.

**Comment by af-doom** [comment]
***@***.*** This is my email address, thank you



---Original---
From: ***@***.***&gt;
Date: Wed, Aug 7, 2024 14:33 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hku-mars/mlcc] The PCD obtained by velodyne16 does not seem toyield good voxels (Issue #23)




    
@Chatoyant19 did you obtain better results?
  
No, I didn't. I know that my problem lies in extracting planar features from the LiDAR point clouds. I 'm unable to extract enough and high-quality planar features.Maybe use PCA to extract planar features is closely related to the quality of point cloud mapping? My LiDAR model is the Hesai FT120, which has relatively poor ranging accuracy.
  
Hello, do you have data about FT120 and camera? Can you share it with me for testing?
  
Of course, can I have your email address? I'll send you the relevant data.

**Comment by Chatoyant19** [comment]
> ***@***.*** This is my email address, thank you
> […](#)
> ---Original--- From: ***@***.***&gt; Date: Wed, Aug 7, 2024 14:33 PM To: ***@***.***&gt;; Cc: ***@***.******@***.***&gt;; Subject: Re: [hku-mars/mlcc] The PCD obtained by velodyne16 does not seem toyield good voxels (Issue #23) @Chatoyant19 did you obtain better results? No, I didn't. I know that my problem lies in extracting planar features from the LiDAR point clouds. I 'm unable to extract enough and high-quality planar features.Maybe use PCA to extract planar features is closely related to the quality of point cloud mapping? My LiDAR model is the Hesai FT120, which has relatively poor ranging accuracy. Hello, do you have data about FT120 and camera? Can you share it with me for testing? Of course, can I have your email address? I'll send you the relevant data. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***&gt;

Sorry, the email information has been encrypted, I cannot access it.
